tests/test_model_upload.py

```diff
@@ -1,246 +1,119 @@
-                self.wfile.write(json.dumps(response).encode("utf-8"))
-        elif self.path == "/api/v1/blobs/upload":
-            KaggleAPIHandler.UPLOAD_BLOB_FILE_NAMES.append(name)
-            self.send_response(200)
-            self.send_header("Content-type", "application/json")
-            self.end_headers()
-            self.wfile.write(
-                json.dumps(
-                    {
-                        "token": "dummy",
-                        "createUrl": "http://localhost:7778",
-                        "status": "success",
-                        "message": "Here is your token and Url",
-                    }
-                ).encode("utf-8")
-            )
-        else:
-            self.send_response(200)
-            self.send_header("Content-Type", "application/json")
-            self.end_headers()
-            error_message = json.dumps({"error": f"bad: {self.path}"})
-            self.wfile.write(bytes(error_message, "utf-8"))
-
-
-class GcsAPIHandler(BaseHTTPRequestHandler):
-    simulate_308 = False
-    put_requests_count = 0
-
-    def do_PUT(self) -> None:  # noqa: N802
-        GcsAPIHandler.put_requests_count += 1
-        if GcsAPIHandler.simulate_308:
-            # Simulate "308 Resume Incomplete" response
-            self.send_response(308)
-            self.send_header("Content-type", "application/json")
-            self.send_header("Range", "bytes=0-499")
-            self.end_headers()
-            GcsAPIHandler.simulate_308 = False
-        else:
-            # Simulate successful upload
-            self.send_response(200)
-            self.send_header("Content-type", "application/json")
-            self.end_headers()
-            self.wfile.write(json.dumps({"status": "success", "message": "File uploaded"}).encode("utf-8"))
 
 
 class TestModelUpload(BaseTestCase):
     def setUp(self) -> None:
-        # Resetting shared variables in GcsAPIHandler
-        GcsAPIHandler.simulate_308 = False
-        GcsAPIHandler.put_requests_count = 0
+        stub.reset()
+
+    @classmethod
+    def setUpClass(cls):  # noqa: ANN102
+        stub.start_server()
 
-        # Resetting any other necessary setup for KaggleAPIHandler
-        KaggleAPIHandler.UPLOAD_BLOB_FILE_NAMES = []
+    @classmethod
+    def tearDownClass(cls):  # noqa: ANN102
+        stub.stop_server()
 
     def test_model_upload_with_invalid_handle(self) -> None:
-        with create_test_http_server(KaggleAPIHandler):
-            with self.assertRaises(ValueError):
-                with TemporaryDirectory() as temp_dir:
-                    test_filepath = Path(temp_dir) / TEMP_TEST_FILE
-                    test_filepath.touch()  # Create a temporary file in the temporary directory
-                    model_upload("invalid/invalid/invalid", temp_dir, APACHE_LICENSE, "model_type")
-                    self.assertEqual(len(KaggleAPIHandler.UPLOAD_BLOB_FILE_NAMES), 1)
```

Comment:
Why did you remove the assertions here?

Reply:
The assertion is that this raises a ValueError. The other statements are not executed.